### ISSUE TITLE
Cropped photo in card mode.

### ISSUE THREAD

**Issue by LazaroFilm** [hariettem/Artemis#362]
**Describe the bug**
Photos are cropped in card mode. 

**To Reproduce**
Browse until you see a vertical photo. If you tap on the image you see it full. Otherwise it's cropped 

**Expected behavior**
See the full picture by default. 
**Screenshots**
![image](https://github.com/hariettem/Artemis/assets/18351100/4bd022e1-c3f4-42c9-aecb-0b2df0f5c54c)

![image](https://github.com/hariettem/Artemis/assets/18351100/9d48d574-86ea-49f7-b155-3e86dbb0bebf)
Notice how the tip and bottom are visible. 

**Smartphone
iPhone 14 pro

Version: 0.2.0

Update: 2308.07.02.24
(f0c15f7c-0d6a-4103-83e7-76ddd674a395)}

Channel: preview

**Comment by RheingoldRiver** [comment]
I do not think this is a bug. @lvndr-dev can you weigh in here?

**Comment by LazaroFilm** [comment]
"It's not a bug, it's a feature..." So can we switch this to a feature request?

**Comment by RheingoldRiver** [comment]
yes, the feature would be to add a preference to see it either way, defaulting to current behavior. but I need confirmation from lavender first before I know which way to proceed (and what priority to give).

when I find out from her we can edit the ticket as needed.

**Comment by LazaroFilm** [comment]
Thanks. I remember that it was an important point about Apollo not cropping pictures like the official app did. I guess the exception was if the picture is taller than the viewport (maybe if it taller, there could be a sign that you're looking at a cropped picture, maybe a colored border on the cropped sides, or a gradient that makes the side of the picture fade out)
